docs/UIDS-GIDS.md

```diff
@@ -57,20 +57,20 @@ Distributions generally split the available UID range in two:
 
 2. 1000…65533 and 65536…4294967294 → Everything else, i.e. regular (human) users.
 
-Note that most distributions allow changing the boundary between system and
-regular users, even during runtime as user configuration. Moreover, some older
-systems placed the boundary at 499/500, or even 99/100. In `systemd`, the
-boundary is configurable only during compilation time, as this should be a
-decision for distribution builders, not for users. Moreover, we strongly
-discourage downstreams to change the boundary from the upstream default of
-999/1000.
+Some older systems placed the boundary at 499/500, or even 99/100,
+and most distributions allow the boundary between system and regular users to be changed
```

Comment:
change "most" to "some". or "traditional". But I#d not make such a quantitative statement here.

Reply:
I think it's important to indicate that at least some distributions allow this to be changed, so that users can take this into account in their mental models of their systems. It's one thing to have a different values on different versions of the system, and another thing to have a different values through local configuration. I changed this to "some", which is vague enough to not be quantitative.